test/integration_test_helper.rb

```diff
@@ -0,0 +1,123 @@
+ENV["RAILS_ENV"] = "test"
```

Comment:
Actually it isn't necessary in neither place, it's only necessary to run `MiniTest::Test` tests, bout our tests inherit from `ActiveSupport::TestCase`. I'll remove it entirely.